### DISCUSSION TITLE
Query on an existing feature

### DISCUSSION THREAD

**Discussion by brundabharadwaj** [codinasion-archive/.github-re#81] [Q&A]
Recently had observed a feature of unassigning issues in  repo if the issue was assigned to a user for more than 15days, Does this feature still exist?

**Comment by harshraj8843** [comment]
Here it is 👉🏻 https://github.com/codinasion/program/actions/workflows/unassign-issues.yml

But, it has to be run `manually`

> **Note** Before running this action please make sure that all PR has been reviewed and `merged`.
>
> Otherwise, if the issue has been assigned for more than 15 days and the linked PR is open, it will unassign the linked issues too.

**Comment by harshraj8843** [comment]
You are already a member of the `program` team. So, you have `write` access to the repo.

It means, in special cases, you can assign issues youself

**Reply by harshraj8843** [reply]
**without using `!assign` command 🤔

**Comment by brundabharadwaj** [comment]
Noted, thank you.
